lib/instruments.js

```diff
@@ -149,8 +150,8 @@ class Instruments {
     this.proc.stderr.setEncoding('utf8');
     let actOnStderr = (output) => {
       if (this.launchTimeout.afterSimLaunch && output && output.match(/CLTilesManagerClient: initialize/)) {
-        this.addSocketConnectTimer(this.launchTimeout.afterSimLaunch, 'afterLaunch', async () => {
-          await killAllInstruments();
+        this.addSocketConnectTimer(this.launchTimeout.afterSimLaunch, 'afterLaunch', async (done) => {
```

Comment:
In origin code, we add the resolve or reject in this.setExitListener and expect after instruments process be killed, it calls this.proc.on('exit', () => {...}) to resolve or reject to exit "await" function -> if the process doesn't be killed, it means we got blocking on that "await"
My solution that we kill instruments (if this proc is exist) and alway call done() to resolve or reject explicitly (avoid blocking in await function).
May my code lacks comments to explain to make you confusing.